src/screens/UserPortal/Events/Events.spec.tsx

```
@@ -69,6 +110,83 @@ vi.mock('react-router', async () => {
+    eventData,
+    viewType,
+  }: {
+    onMonthChange?: (month: number, year: number) => void;
+    eventData?: unknown[];
+    viewType?: string | null;
+  }) => (
+    <div>
+      <button
+        data-testid="monthChangeBtn"
+        onClick={() => onMonthChange && onMonthChange(5, 2023)}
+      />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)